frc-docs-examples/ctre-drivetrain/robot.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+"""
+    PWM Drivetrain Demo
+"""
+
+# The first lines should be the import of the needed modules
+import wpilib
+import wpilib.drive
+import ctre
+
+
+# All code should be found in the MyRobot class. Note that in this example, the wpilib.TimedRobot robot base is used.
+class MyRobot(wpilib.TimedRobot):
+    def robotInit(self):
+        """Robot initialization function"""
+
+        # create motor controller objects
+        m_left = ctre.WPI_TalonFX(1)
```

Comment:
This isn't a PWM drivetrain? What's the original example that we're referencing here?

Reply:
The idea is to replicate the same examples as in this frc-docs section:

https://docs.wpilib.org/en/stable/docs/zero-to-robot/step-4/creating-test-drivetrain-program-cpp-java.html#defining-the-variables-for-our-sample-robot

There are simple drivetrain examples for various vendors

Reply:
In the source for those (https://github.com/wpilibsuite/frc-docs/blob/stable/source/docs/zero-to-robot/step-4/creating-test-drivetrain-program-cpp-java.rst), the CTRE/REV examples aren't in the allwpilib repo, but are directly in the frc-docs source code.

I've been trying to avoid putting vendor-specific examples in our example repo, choosing to put them in the examples folder for each individual vendor (and the README for this repo says "These examples are for core RobotPy/WPILib libraries. Examples specific to individual vendors can be found in their individual RobotPy repository under their 'examples' folder").

This doesn't seem like a good reason to change yet -- if you wanted to use a remoteliteralinclude in frc-docs, you could point at the robotpy-ctre/robotpy-rev repositories instead and put these examples there. Happy to merge them over there.